internal/mutators/podtemplatespec.go

```diff
@@ -0,0 +1,118 @@
+package mutators
+
+import (
+	"fmt"
+
+	"github.com/kanopy-platform/hedgetrimmer/internal/limitrange"
+	"github.com/kanopy-platform/hedgetrimmer/internal/quantity"
+	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+type PodTemplateSpec struct {
+	defaultMemoryLimitRequestRatio float64
+}
+
+func NewPtsMutator(defaultMemoryLimitRequestRatio float64) PodTemplateSpec {
```

Comment:
```suggestion
func NewPodTemplateSpec(defaultMemoryLimitRequestRatio float64) PodTemplateSpec {
```

If its desired to include the `Mutator` suffix, the struct should be renamed to `PodTemplateSpecMutator`